src/GeositeFramework/js/Plugin.js

```diff
@@ -32,7 +32,40 @@ require(['use!Geosite',
                 pluginName = model.get('pluginSrcFolder'),
                 $uiContainer = model.get('$uiContainer'),
                 $legendContainer = model.get('$legendContainer'),
-                logger = new Logger(pluginName);
+                logger = new Logger(pluginName),
+                resizers = {
+                    adjustBy: function(val) {
+                        var currentWidth = parseFloat($uiContainer.css("width"));
+                        var d = $.Deferred();
```

Comment:
Consider using the Dojo Deferred class. The docs also have some good example code to demonstrate how deferred objects should be used.

Ref: http://dojotoolkit.org/reference-guide/1.9/dojo/Deferred.html